src/Reveal.Sdk.Dom.Tests/Data/DataSourceItems/AmazonAthenaDataSourceItemFixture.cs

```diff
@@ -0,0 +1,85 @@
+using Reveal.Sdk.Dom.Data;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSourceItems
+{
+    public class AmazonAthenaDataSourceItemFixture
+    {
+        [Fact]
+        public void Constructor_WithDataSource_SetsTitleAndDataSource()
+        {
+            // Arrange
+            string title = "Test Item";
+            var dataSource = new AmazonAthenaDataSource();
+
+            // Act
+            var item = new AmazonAthenaDataSourceItem(title, dataSource);
```

Comment:
Initializing an object instance should be in Arrange. Act only includes the method calling / state updating.